.nvmrc

```
@@ -1 +1 @@
-v10.8.0
+v12.18.0
```

I think for the starter we should pin this to something less specific I kind of like `lts/*` but we could do `12.18` or even just `12` too.
```suggestion
lts/*
```

`lts/*` will pin this to whatever version of node is in long term stable, which is almost always what we want, especially since this build isn't super complex and we're relatively unlikely to run into version issues (as long as we keep the packages updated)

---

TIL!